src/main/java/hudson/plugins/cobertura/Ratio.java

```diff
@@ -61,7 +69,7 @@ public float getPercentageFloat() {
      * @return percentage
      */
     public String getPercentageString() {
-      return dataFormat.format(getPercentageFloat());
+        return dataFormat.format(getPercentageFloat());
```

Comment:
The Jira ticket asked to `render anything between 99% (or 99.5%) and 100% as 99+% (or similar)`. Could you do that here? It would be good to be able to see the difference between actual 99% and something rounded down to 99%

Reply:
This method is used for sort table. But in `sortable.js`, it determined **limited** sorter by using regex. And If I add a character that is not number but some special character like `+`, the table will be sorted by comparing string instead of the original behavior. This return value will only be used by js code and not visible by the user(If the user doesn't view HTML content directly). 
So I think to diff the difference by simply rounding down number larger than 99.995 to three decimal places, rounding off number less than 99.995 to two decimal places is ok. And what do you think?

Reply:
Yeah, that makes sense. I posted a comment on the Jira ticket asking for feeback from the original reporter. If they don't strongly object pretty quickly, I'll accept this PR (with the additional tests of course). As a Cobertura user, I'd be happy with it - we only provide single accuracy to 1% anyway, so adding some + character doesn't convey much additional information.